etc/config/analysis.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&llvm-mca
+compilerType=analysis
+supportsBinary=false
+objdumper=
+demangler=
+
+group.llvm-mca.compilers=llvm-mca
```

Comment:
Would be great if you added the version to the compiler key. (`llvm-mcatrunk` for example)

Reply:
Done